js/bybit.js

```diff
@@ -4515,29 +4515,19 @@ module.exports = class bybit extends Exchange {
     async fetchDerivativesOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
         await this.loadMarkets ();
         let market = undefined;
-        let settle = undefined;
         const request = {
             // 'symbol': market['id'],
-            // 'order_id': 'string'
-            // 'order_link_id': 'string', // unique client order id, max 36 characters
-            // 'symbol': market['id'], // default BTCUSD
-            // 'order': 'desc', // asc
-            // 'page': 1,
-            // 'limit': 20, // max 50
-            // 'order_status': 'Created,New'
-            // conditional orders ---------------------------------------------
-            // 'stop_order_id': 'string',
-            // 'stop_order_status': 'Untriggered',
+            // 'orderId': 'string'
+            // 'orderLinkId': 'string', // unique client order id, max 36 characters
+            // 'orderStatus': 'Created,New'
+            // 'orderFilter': 'StopOrder', // 'Order' or 'StopOrder'
+            // 'limit': 20, // Limit for data size per page. [1, 50]. Default: 20
+            // 'cursor': 'string', // used for pagination
         };
         if (symbol !== undefined) {
             market = this.market (symbol);
-            settle = market['settle'];
```

Comment:
Interestingly, this method was working on my side even with the settleCoin, but ofc we should fix it